src/components/Header.jsx

```diff
@@ -1,37 +1,45 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import user from '../assets/user.svg';
 
 function Header() {
   return (
     <nav className="navbar navbar-expand-sm navbar-light bg-white">
       <div className="container">
-        <Link className="navbar-brand" to="/">
-          Bookstore CMS
-        </Link>
-        <button
-          className="navbar-toggler"
-          type="button"
-          data-bs-toggle="collapse"
-          data-bs-target="#navbarNav"
-          aria-controls="navbarNav"
-          aria-expanded="false"
-          aria-label="Toggle navigation"
-        >
-          <span className="navbar-toggler-icon" />
-        </button>
-        <div className="collapse navbar-collapse" id="navbarNav">
-          <ul className="navbar-nav ms-auto">
-            <li className="nav-item">
-              <Link className="nav-link" to="/">
-                Books
-              </Link>
-            </li>
-            <li className="nav-item">
-              <Link className="nav-link" to="/categories">
-                Categories
-              </Link>
-            </li>
-          </ul>
+        <div className="d-flex align-items-sm-baseline">
+          <Link className="navbar-brand" to="/">
+            <h2 className="text-primary fw-bolder">Bookstore CMS</h2>
+          </Link>
+          <button
+            className="navbar-toggler"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarNav"
+            aria-controls="navbarNav"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span className="navbar-toggler-icon" />
+          </button>
+          <div className="collapse navbar-collapse" id="navbarNav">
+            <ul className="navbar-nav ms-auto">
+              <li className="nav-item">
+                <Link className="nav-link" to="/">
+                  <p>Books</p>
+                </Link>
+              </li>
+              <li className="nav-item">
+                <Link className="nav-link" to="/categories">
+                  <p>Categories</p>
+                </Link>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div className="oval">
+          <button type="submit" className="btn btn-sm">
+            <img src={user} alt="" />
+          </button>
         </div>
```

Comment:
- [ ] According to the project requirement Styling is **EXACTLY** the same as the one provided on Zeplin. So Please see the design below and try to use the same design. Also, try to use the same color. Everything is provided there color, font etc.
### Your design
![his](https://github.com/PabloBona/bookstore-react-app/assets/81727988/e2b9b18c-f00b-4441-8fbb-60b3f07cd1b3)

### Original design
![original](https://github.com/PabloBona/bookstore-react-app/assets/81727988/b74590f7-a615-4289-9d7f-340387e3886f)